CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ option(GO_USE_JACK	      "Use native Jack output" ON)
 if (WIN32 OR APPLE)
    option(INSTALL_DEPEND      "Copy dependencies (wxWidgets libraries and Translations) on installation" ON)
 else ()
-   option(INSTALL_DEPEND      "Copy dependencies (wxWidgets libraries and Translations) on installation" OFF)
+   option(INSTALL_DEPEND      "Copy dependencies (wxWidgets libraries and Translations) on installation" ON)
```

Comment:
Also, you shouldn't need to switch on copy dependencies when building on Linux...

Reply:
Yes, it was just a test.

But I finally succeeded in using :

`include(UsewxWidgets)`

Reply:
Now, I have a problem including core libraries to get usable classes in main test file

Reply:
Ok, I got it. I needed to use target_include_directories() to ... include header files.
So, now ready to dig into code :smiley: